socialmedia.md

```diff
@@ -1,20 +1,118 @@
+
+#### Snapchat
+* Mission: Life’s more fun when you live in the moment.
+* Audience: Anyone interested in your school specifically – more likely to be parents, community, pupils (dependent on age restrictions).
+* Recommended Use: Have a school account and capture exciting moments as they happen – photo and video.
+
+#### Pinterest
+* Mission: The world’s catalogue of ideas.
+* Audience: Parents and other teachers looking for inspiration.
+* Recommended Use: Have a school account and create a board for each topic or subject. Pin ideas to the board for other teachers to use, or for parents to see as homework/out of school ideas. Invite your teachers to contribute to the board.
+
+#### Youtube
+* Mission: A forum for people to connect, inform, and inspire others across the globe. Videos, trends and channels.
+* Audience: Anyone interested in your content or theme, i.e. education, school, teaching, children.
+* Recommended Use: To share videos that promote your school and learning environment. Christmas videos, shows, assemblies.
+
+
+### Our Social Media Accounts
+
+The school will have a presence on the following social networks: __Facebook, Twitter, LinkedIn, Instagram, Pinterest, Snapchat and Youtube__. Where possible, the account will be listed as a business or company under the school’s name, rather than as a personal or individual. It is important for us, as a school, to maintain a presence on these platforms to discourage others from creating a derogatory account in our name.
+
+The school will actively post on the following social networks: __Facebook, Twitter and Instagram.__
+Employees at the school are permitted to have personal accounts on any social networks they choose, providing they follow the rules outlines in Part 2 of this document and do not attempt to impersonate the school through an unofficial account.
+
+#### Links to Our Accounts
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+### General Rules and Restrictions
+
+#### Posting on Social Media
+* Content will be posted to official school pages only.
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
+* Any photographs or videos taken within school grounds, or during school time, should only be shared to a school account. 
+
+#### Photographs and Video
+* Media will be posted to school accounts only.
+* If the school keep a list of children who do not have parental consent to be online, this should be cross referenced before every post (and an audit trail of this kept for future reference).
```

Comment:
'Must Be' not 'should'. - If the suggestion is an audit trail, in what form would this be? how could/should it be implemented.  - The Audit trail might need dropping in all fairness.